mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -294,11 +280,19 @@ class MetalsGlobal(
       isVisited += key
       val result = tpe match {
         case TypeRef(pre, sym, args) =>
-          def backtickifiedSymbol = backtickify(sym)
+          def shortSymbol = {
+            /* If it's an alias type we want to prevent dealiasing it
+               AnyRef should stay to be dropped if neded later on since it's
+               not an important class.
+             */
+            if (sym.isAliasType && sym != definitions.AnyRefClass)
+              backtickify(sym.newErrorSymbol(sym.name).updateInfo(sym.info))
```

Comment:
This makes sure that sym is not dealiased ince the error type is not an alias, but we still have all the correct type info.